### DISCUSSION TITLE
Agent Names in picklists are not alphabetized correctly

### DISCUSSION THREAD

**Discussion by yvonneekanim** [specify/specify7#5530] [Feature Requests]
Reported by Tomislav:

It seems that the alphabetization on the agent list is not quite right. We have a student with a common last name who has to look themselves up on the cataloger field (see above) and the alphabetization is not behaving as expected. Here is an example:

 
![image001](https://user-images.githubusercontent.com/28809154/68968628-ab6a6c00-07a8-11ea-877f-32437bf15afa.jpg)

**Comment by maxpatiiuk** [comment]
This should be fixable by the collate MySQL setting:
https://www.geeksforgeeks.org/what-is-collation-and-character-set-in-mysql/

**Comment by carlosmbe** [comment]
Was able to recreate a similar sorting pattern. Going to investigate the collate MySQL settings and will report back 
<img width="1201" alt="Screenshot 2024-01-04 at 12 26 14 PM" src="https://github.com/specify/specify7/assets/53784701/eb311862-6e89-4929-a47f-eec719e3a22f">

**Comment by carlosmbe** [comment]
The collation settings for the server, connection, and database are all `utf8mb4_general_ci`

The database and "Tomislav" are both from UT Austin for your reference. Their data is not conveniently available and is in the SCC Vault.

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6), it's not very consistent but it seems to have something to do with spaces and special characters but I'm not entirely sure
